create_package.py

```diff
@@ -186,9 +186,7 @@ def zip_client_side(addon_package_dir, current_dir, log):
     """
 
     client_dir = os.path.join(current_dir, "client")
-    if not os.path.isdir(client_dir):
-        log.info("Client directory was not found. Skipping")
-        return
+    addon_subdir_path = os.path.join(client_dir, ADDON_CLIENT_DIR)
```

Comment:
I assume we check somewhere else that client dir exists

Reply:
Nope, that was "universal" code for base addons, in shotgrid case we 100% know there should be client dir...